### DISCUSSION TITLE
Broken link in password recovery: user locked out for good

### DISCUSSION THREAD

**Discussion by johann-petrak** [npm/feedback#689] [Website] [answered]
I wrongly thought I had the wrong password because the login-form does not accept the email in place of the user name (which is bad)

I requested a password reset, got the temporary password, had then to wait for my OTP, entered the OTP and after entering this,
the web site showed an "unknown link" 401 error: https://www.npmjs.com/login/email-otp?next=%2F
without me being logged in.

So there is no way at all to login via the web.

I was trying to contact support via email, was told this is not possible, only via the web, but via the web one has to be logged in!
How is one supposed to get support for login problems if a login is required?
The whole way how NPM security and support does not work is really really frustrating and already made me waste a lot of time.

**Comment by monishcm** [comment]
I tried reproducing this, apparently works fine for me. I also tried to check if you had opened a support ticket but could not find any. You can open a support ticket at [http://npmjs.com/support](http://npmjs.com/support) without being logged in and we will check your issue.

**Comment by johann-petrak** [comment]
Did you reproduce this using the process of resetting the password? I just retried and it still does not work. Here is the complete sequence of steps:

* Goto https://www.npmjs.com/login
* Try to log in with existing password: does not work
* Click "Forgot password"
* Enter my email and click "email me a recovery link"
* Receive the email and click the link in it
* Click "Continue"
* Copy the password, then click "Login"
* In the login dialog, enter my user name paste the copied password
* Click Sign In
* get the "Login Veification" dialog https://www.npmjs.com/login/email-otp?next=%2F
* Wait for the OTP by email, copy the OTP and enter into the login verification dialog
* After this I get an error screen with the error messages:

```
"not found"
"Route not found!"
"404 HAVE YOU SEEN THIS WOMBAT"
The URL of this page is https://www.npmjs.com/login/email-otp?next=%2F
```

**Reply by monishcm** [reply]
I did the same thing and seemed to work. Can you please open a support ticket @ npmjs.org/support with these steps? We will investigate if we are hitting any edge case and take action to get back your account. Appreciate your patience in detailing it out.

**Comment by WLyKan** [comment]
I have this `Route not found!` error too!

![image](https://user-images.githubusercontent.com/2155093/170436483-71c8a6dc-888a-460a-b4d7-389031b56e24.png)

![image](https://user-images.githubusercontent.com/2155093/170436526-8bc9427f-d12d-47c8-90cf-d34c954edeb1.png)

**Comment by 42arch** [comment]
I got this error when I use my pc to log in. Then I use my cellphone (with the cellular network) to do this again, and it works! Maybe it's caused by the network?

**Comment by johann-petrak** [comment]
After 13 days I have not heard from support nor has this been fixed and I am still locked out of my account. 
My trust in NPM is approaching 0.

**Reply by MylesBorins** [reply]
Hey @johann-petrak I'm digging into this on our end. Engineering wasn't able to find a reason why this would be happening. Have you tried clearing browser cache and using incognito mode or a different browser?

We had some issues with reset last week that got fixed, hopefully it works now after taking the above steps.

**Comment by johann-petrak** [comment, marked as answer]
In my case and maybe also helpful for others: changing the password and also logging in worked fine in another browser profile (same browser version: Firefox 100.0.2). 
After some experimentation in my original browser profile it turned out that disabling my Firefox extension "uBlock Origin" for the site www.npmjs.com made the problem go away and I was able to login.  Weirdly though, after re-enabling it the login still works. Weird.
